src-built-in/components/advancedAppLauncher/src/components/FoldersList.jsx

```diff
@@ -214,10 +214,16 @@ export default class FoldersList extends React.Component {
 
 		const EDITABLE_FOLDER_ICON_CLASS = 'ff-adp-hamburger'
 
-		let nameField = folder.icon === EDITABLE_FOLDER_ICON_CLASS && this.state.renamingFolder === folderName ?
-			<input id="rename" value={this.state.folderNameInput}
-				onChange={this.changeFolderName}
-				onKeyPress={this.keyPressed} className={this.state.isNameError ? "error" : ""} autoFocus /> : folderName;
+		let nameField;
```

Comment:
It looks like you've changed the logic here by adding a case: `folderName === "Advanced App Launcher"`. Can you explain this change?

Reply:
I confirmed this with Josh and Brad: we want the UI to show "App Launcher" even if users are using advanced app launcher

Reply:
Ok - makes sense then.